lib/Core/Executor.cpp

```diff
@@ -2998,12 +2977,33 @@ void Executor::run(ExecutionState &initialState) {
     }
 
     klee_message("seeding done (%d states remain)", (int) states.size());
+    klee_message("<CONC> sizeof prio2: %lu", prio2.size());
```

Comment:
I'm curious why we're using a priority list f we're dumping these after we've replayed the seeds. I can see it may be useful later, though.

Reply:
Yes, the test cases will only be dumped after replay. The idea is that the solver should only be involved in order to follow the branches as determined by the seed during replay, and when replay is done, it should only try to flip those branches corresponding to the states in prio2, then prio1, and not try to solve prio0 branches at all. This is probably not the ideal/right way to do it though?

Reply:
I'm wondering what we lose if we dump the states during the replay (as in my original code) vs waiting till the end. If there's a good reason to defer, that's fine.

Reply:
Currently it is not very useful indeed, so I will revert to your original code that dumps the states during replay. In the long run, if we find a way to pass a list of edges that the fuzzer has already covered to KLEE, we would not have to spend time to flip a branch corresponding to a covered edge, so this branch would be considered prio0.